spec/lib/vmdb/config_spec.rb

```diff
@@ -61,6 +61,30 @@
       expect(SettingsChange.first).to have_attributes(:key => '/api/token_ttl', :value => "1.day")
     end
 
+    it "saves zone config" do
+      resource = FactoryGirl.create(:zone)
+      MiqRegion.seed
+      data = {}
+      data.store_path(:api, :token_ttl, "1.day")
+      data = data.to_yaml
+
+      expect(VMDB::Config.save_file(data, resource)).to be true
+      expect(SettingsChange.count).to eq(1)
+      expect(SettingsChange.first).to have_attributes(:key => '/api/token_ttl', :value => "1.day")
```

Comment:
We should add a test that also shows this change is attached to the zone...something like `expect(SettingsChange.first.resource).to eq(resource)`.  This should be done for all 3 of the server/zone/region tests